modules/nvidia-gpu-azure-adding-a-gpu-node.adoc

```diff
@@ -0,0 +1,435 @@
+// Module included in the following assemblies:
+//
+//  * machine_management/creating-machinesets/creating-machineset-azure.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-aws-adding-a-gpu-node_{context}"]
+= Adding a GPU node to an existing {product-title} cluster
+
+You can copy and modify a default compute machine set configuration to create a GPU-enabled machine set and machines for the Azure cloud provider.
+
+The following table lists the validated instance types:
+
+[cols="1,1,1,1"]
+|===
+|vmSize |NVIDIA GPU accelerator |Maximum number of GPUs |Architecture
+
+|`Standard_NC24s_v3`
+|V100
+|4
+|x86
+
+|`Standard_NC4as_T4_v3`
+|T4
+|1
+|x86
+
+|`ND A100 v4`
```

Comment:
@fabiendupont @wabouhamad Please provide the "NVIDIA GPU accelerator" and "Architecture" for ND A100 v4.

Reply:
@StephenJamesSmith "NVIDIA GPU accelerator" is "A100" and Architecture is "x86".

Reply:
Added.